quickstart_guides/ruby.md

```diff
@@ -2,3 +2,84 @@ address: support.aptible.com/quickstart/ruby
 
 ### Ruby Quickstart
 
+This guide assumes you have:   
+- An Aptible account,  
+- An SSH key associated with your Aptible user account, and  
+- The Aptible command line tool installed
+
+#### 1. Provision Your App  
+Tell the Aptible API that you want to provision an application. Until you push code and trigger a build, Aptible uses this as a placeholder.
+
+##### Command line
+Use the `apps:create` command: `aptible apps:create HANDLE`
+
+For example: 
+```
+aptible apps:create my-example-app
+```
+
+If you have multiple accounts, you will need to specify `--account <account_handle>`. Each account handle is listed next to the account name in the Aptible Dashboard.
+
+For example: 
+```
+aptible apps:create example-app --account myco-development
+```
+
+##### Aptible Dashboard  
+From the [Aptible dashboard](https://dashboard.aptible.com/apps), pick an account to provision the app into, select "Create App," and choose a handle.
+
+#### 2. Add a Dockerfile
```

Comment:
Sort of on the fence about this but tend to agree with both of you that it would be an easy feature to add and would make it much easier for the average user just trying to get things up and running. My only concern is that for a large percentage of users coming from non-vanilla Heroku environments I'd expect a buildstep Dockerfile to not work out-of-the-box because it'll lack some dependency they need: a mysql-client, python-dev, etc. and those people will skip over the Dockerfile and run into errors, then we'll have to back them out of buildstep and help them put together a custom Dockerfile. We might end up causing more grief than we solve this way.